rs/sns/governance/canister/canister.rs

```diff
@@ -733,6 +734,14 @@ fn add_maturity_(request: AddMaturityRequest) -> AddMaturityResponse {
     governance_mut().add_maturity(request)
 }
 
+#[cfg(feature = "test")]
+#[export_name = "canister_update advance_target_version"]
+fn advance_target_version(request: AdvanceTargetVersionRequest) {
+    over(candid_one, |request: AdvanceTargetVersionRequest| {
+        AdvanceTargetVersionResponse {}
```

Comment:
Why not implement this function in the same PR?

Reply:
That is the plan, however the scope of this PR has unfortunately grown and I may have to start splitting it up before merging it

Reply:
Yes, let's keep this PR just for bumping the target in testing. 

Note that it should still only be possible to advance the target using new_target, not go back to an older version (I recommend implementing that logic in this PR as well).